docs/upgrade-pro.md

```diff
@@ -0,0 +1,61 @@
+# Upgrade to Percona Server for MySQL Pro
+
+Are you a Percona Customer already and are you ready to enjoy all the [benefits of Percona Server for MySQL Pro](../psmysql-pro.md)? 
+
+This document provides instructions on how you can upgrade from Percona Server for MySQL Basic to Percona Server for MySQL Pro.
+
+## Preconditions 
+
+Request the access to the pro repository from Percona Support. You will receive the client ID and the access token.
```

Comment:
client ID and access token which you use when downloading the packages.

Reply:
Updated